scripts/deploycdn.sh

```diff
@@ -0,0 +1,26 @@
+# Install blobxfer
+curl -L -o ~/blobxfer https://github.com/Azure/blobxfer/releases/download/1.2.1/blobxfer-1.2.1-linux-x86_64
+chmod +x ~/blobxfer
+
+PACKAGE_NAME=$(node -p require\(\'$TRAVIS_BUILD_DIR/package.json\'\).name)
+PACKAGE_VERSION=$(node -p require\(\'$TRAVIS_BUILD_DIR/package.json\'\).version)
+echo Will publish to CDN at $PACKAGE_NAME/$PACKAGE_VERSION/*
+
+# Copy files that should be distributed to CDN
+mkdir $TRAVIS_BUILD_DIR/dist
+
+cp $TRAVIS_BUILD_DIR/adaptivecards-hostconfig.json $TRAVIS_BUILD_DIR/dist
+cp $TRAVIS_BUILD_DIR/botchat-fullwindow.css $TRAVIS_BUILD_DIR/dist
+cp $TRAVIS_BUILD_DIR/botchat.css $TRAVIS_BUILD_DIR/dist
+cp $TRAVIS_BUILD_DIR/botchat.js $TRAVIS_BUILD_DIR/dist
+cp $TRAVIS_BUILD_DIR/CognitiveServices.js $TRAVIS_BUILD_DIR/dist
```

Comment:
If its bash, this should work.
sources=(
        adaptivecards-hostconfig.json
        botchat-fullwindow.css
        botchat.css
        botchat.js
        CognitiveServices.js
)

for i in ${sources[*]}
do
        echo cp $TRAVIS_BUILD_DIR/$i $TRAVIS_BUILD_DIR/dist
done

Reply:
Fixed.